common/cftoFunctions.js

```diff
@@ -45,18 +48,44 @@ function chooseCollection() {
     browser.click('.collection--summary__btn');
 }
 
+function chooseCollectionIE() {
+    //Select store from the store list and confirm
+    browser.click('#store_4048');
+    browser.click('.btn--primary');
+
+    //Confirm the store selection date
+    browser.click('.btn--primary');
+
+    //Select the store selection time slot
+    browser.click('.collection-continue .btn--primary');
+
+    //Click to confirm the choosen collection slot data & move-on to payment page
+    browser.click('.collection--summary__btn');
+}
+
 function cardDetailsLightBox() {
     browser.selectByValue('select[name="cardType"]', '11105_Mastercard');
-    browser.setValue('#account', '5105105105105100');
+    browser.setValue('#account', '5454609899026213');
```

Comment:
This is a Test Card only. Actually the previous card was throwing frequent errors. So, switch it to a more reliable test card